frontend/src/components/GoalsTable/GoalRow.js

```diff
@@ -203,24 +205,28 @@ function GoalRow({
 
   const displayGoalTopics = truncateGoalTopics(goalTopics);
 
+  const expandObjectives = () => {
+    setObjectivesExpanded(!objectivesExpanded);
+  };
+
   return (
-    <tr className="tta-smarthub--goal-row" key={`goal_row_${id}`}>
-      <td>
-        {getGoalStatusIcon()}
-        {displayStatus === 'Ceased/suspended' ? ['Ceased/ ', <br />, 'suspended'] : displayStatus}
-      </td>
-      <td>{moment(createdOn).format(DATE_DISPLAY_FORMAT)}</td>
-      <td className="text-wrap maxw-mobile">
-        {goalText}
-        {' '}
-        (
-        {goalNumber}
-        )
-        {determineFlagStatus()}
-      </td>
-      <td className="text-wrap maxw-mobile">
-        {
-          showToolTip
+    <>
+      <tr className={`tta-smarthub--goal-row ${!objectivesExpanded ? 'tta-smarthub--goal-row-collapsed' : ''}`} key={`goal_row_${id}`}>
+        <td>
+          {getGoalStatusIcon()}
+          {displayStatus === 'Ceased/suspended' ? ['Ceased/ ', <br />, 'suspended'] : displayStatus}
+        </td>
```

Comment:
Suspended or Ceased/Suspended? Also not sure about the way its wrapping around the icon, looks a little funky to me

Reply:
I will switch to just 'Suspended' as this was what was decided.